### PR TITLE
[Multi] Add useUpdateCreator hook, fix CreatorInput schema

### PR DESCRIPTION
**Pull request by nachoiacovino**

### CODE REVIEW

File: packages/sdk/src/solana/classes/helpers/creators-helper.ts [hunk 1]
```diff
@@ -9,18 +9,19 @@ export function enforceCreator(
   creators: CreatorInput[] = [],
   owner: PublicKey,
 ): Creator[] {
+  console.log("enforceCreator", creators, owner);
```
Comment (joaquim-verges) [inline packages/sdk/src/solana/classes/helpers/creators-helper.ts:12]: remove log

File: packages/sdk/src/solana/types/programs/index.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const CommonContractOutputSchema = CommonContractSchema.extend({
  */
 export const CreatorInputSchema = z.object({
   address: z.string(),
-  sharePercentage: z.number(),
+  share: z.number(),
```
Comment (joaquim-verges) [inline packages/sdk/src/solana/types/programs/index.ts:30]: only annoying thing is that now its not clear if the unit is bps or percentage. Maybe we can add a transform here to ensure the value is between 0 and 100 ? you can copy what we did for BasisPointsSchema

Comment (nachoiacovino) [reply packages/sdk/src/solana/types/programs/index.ts]: used the PercentSchema we already had

**Review by joaquim-verges — APPROVED** [review]
lg

**Pull request merged**